articles/tokens/guides/rotate-refresh-tokens-sdk.md

```diff
@@ -0,0 +1,43 @@
+---
+description: Learn how to rotate fresh tokens in SPAs using the auth0-spa-js.
+topics:
+  - tokens
+  - refresh-token-rotation
+contentType: how-to
+useCase:
+  - refresh-token-rotation
+  - silent-authentication
+---
+# Rotate Refresh Tokens in SPAs Using auth0-sp-js SDK
+
+To take advantage of the Refresh Token rotation functionality, the option `useRefreshTokens` has been added to `createAuth0Client`, which defaults to `false`.
+
+* With this option set to `true`, the `offline_access` scope is automatically requested when using `loginWithRedirect(), loginWithPopup()` and `getTokenSilently()`. Furthermore, when `getTokenSilently()` is invoked and the access token has expired, the SDK will attempt to renew the ID and Access Tokens by calling the `/token` endpoint using the `refresh_token` grant type along with the Refresh Token from the cache. 
```

Comment:
If this exchange fails, what do we do @stevehobbsdev ?

Reply:
It depends on at what point it fails.

If the exchange effectively fails because we don't have a refresh token, then we fall back to the iframe method and try that (which would in turn fail if third-party cookies are blocked).

If there is a failure actually calling the `/token` endpoint with a refresh token, we would bubble that up to the developer.

Reply:
Added clarification